buildtest/buildsystem/base.py

```diff
@@ -318,36 +335,23 @@ def get_environment(self):
         """
 
         env = []
-        pairs = self.recipe.get("env")
-        shell = self.get_shell()
-
-        # If we are using a python shell, we need to import os first
-        if "python" in shell:
-            env.append("import os")
+        pairs = self.recipe.get("env", [])
+        shell = self.shell.name
 
         # Parse environment depending on expected shell
-        if pairs:
-
-            # Handles bash and sh
-            if re.search("(bash|sh)$", shell):
-                [env.append("%s=%s" % (k, v)) for k, v in pairs.items()]
+        # if pairs:
```

Comment:
This is commented out and should not be there.

Reply:
See https://github.com/buildtesters/buildtest/pull/290#commitcomment-38882276. I introduced this bug by commenting this out. I guess we still need the condition ``if pairs`` when a Buildspec doesn't define ``env`` section we just skip the whole thing